content/nextflow-on-aws-batch/30_batch/00_ami.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Create Custom AMI"
+chapter: false
+weight: 01
+hidden: true
```

Comment:
yes, but I wanted to keep it around. encapsulating everything in a container is the ultimate rainbow, unicorn dream. 
For portability reasons it might be wise to tweak the AMI.